hazelcast-client-new/src/main/java/com/hazelcast/client/proxy/ClientMapProxy.java

```diff
@@ -527,6 +543,13 @@ public String addEntryListener(MapListener listener, boolean includeValue) {
     }
 
     @Override
+    public String addEntryListener(EntryListener listener, boolean includeValue) {
```

Comment:
https://hazelcast-l337.sonar.cloudbees.com/drilldown/measures/10815?metric=coverage&rids%5B%5D=10817&rids%5B%5D=13439#   sonar seems to show this function is not covered by any tests ?